.env.example

```diff
@@ -62,3 +62,4 @@ MATTERS_ALCHEMY_API_KEY=
 MATTERS_OPENSEA_API_KEY=
 MATTERS_POLYGON_CURATION_CONTRACT_ADDRESS=0xa219C6722008aa22828B31A13ab9Ba93bB91222c
 MATTERS_POLYGON_CURATION_CONTRACT_BLOCKNUM=28675517
+MATTERS_EXCHANGE_RATES_DATA_API_KEY=QLqGNNhnz8MHpwOh9AqTJv069po2aR09
```

Comment:
We are using Open Exchange Rates API as HKD to USD conversion for payout, is there any way to combine it with this? 

https://github.com/thematters/matters-server/blob/develop/src/connectors/paymentService.ts#L623-L629

Reply:
Open Exchange Rates API free plan only support USD related exchange rates querying and thus we should reuse `ExchangeRate.getRates` here

Reply:
Yeah, let's see if we should change it in this PR.